src/test/java/models/CustomerModel.java

```diff
@@ -0,0 +1,43 @@
+package models;
+
+import com.github.javafaker.Faker;
+import lombok.*;
+
+import java.util.Locale;
+
+@Getter
+@Setter
+
```

Comment:
Remove empty line here. Use @Builder instead of setter and make all fields final.

Reply:
You can also use:
@Getter
@Builder
@FieldNameConstants
@RequiredArgsConstructor